java/code/src/com/redhat/rhn/domain/server/MinionServer.java

```diff
@@ -218,6 +218,10 @@ private boolean isAmazon2() {
         return ServerConstants.AMAZON.equals(getOsFamily()) && getRelease().equals("2");
     }
 
+    private boolean isRocky8() {
+        return ServerConstants.ROCKY.equals(getOs()) && getRelease().equals("8");
```

Comment:
I think using `getOsFamily()` here was what caused the problem with the product detection. It's `RedHat` for Rocky Linux.

Reply:
Second problem: `getRelease().equals("8");` does not work, as it seems the release is `8.4' right now. Changed to match `8.*`